torch2trt/converters/getitem_test.py

```diff
@@ -155,11 +155,39 @@ def forward(self, x):
 
     data = torch.randn(1, 3, 112, 112).cuda()
     assert(torch.allclose(module_trt(data), module(data), atol=1e-4, rtol=1e-4))
-    
+
     data = torch.randn(4, 3, 112, 112).cuda()
     assert(torch.allclose(module_trt(data), module(data), atol=1e-4, rtol=1e-4))
 
 
-if __name__ == '__main__':
+def test_getitem_dynamic_gathernd():
+
+    class TestModule(nn.Module):
+        def forward(self, tensor, indices):
+            #  indices = indices + indices
+            return tensor[:, indices.to(torch.int64), (1, 0, 1)]
+
+    module = TestModule().cuda().eval()
 
-    test_getitem_dynamic()
\ No newline at end of file
+    tensor = torch.rand(3, 5, 4, 5).cuda()
+    indices = torch.tensor((2, 0, 1), dtype=torch.int32).cuda()
+
+    module_trt = torch2trt(module, [tensor, indices], min_shapes=[(1, 1, 1, 1), (1, 1, 1)], max_shapes=[(7, 7, 7, 7), (5, 5, 5)], log_level=trt.Logger.INFO)
+
+    assert torch.allclose(module_trt(tensor, indices), module(tensor, indices), atol=1e-4, rtol=1e-4)
+
+    tensor = torch.rand(2, 4, 3, 4).cuda()
+    indices = torch.tensor((2, 0, 1), dtype=torch.int32).cuda()
```

Comment:
@jaybdub I think `indices` here could also be made to support dynamic values, but the current iteration of this isn't working... seems like the input flattener is removing this tensor from `inputs` before the `add_inputs` call for some reason (so we only see one input tensor instead of two). I'll have to dig into this more later.